src/foam/dao/EasyDAO.js

```diff
@@ -415,6 +423,14 @@ return delegate;
             src: dao,
             of: this.model});
         }
+
+        if ( this.cacheType == 'LRU' ) {
+          this.mdao = this.MDAO.create({of: params.of});
+          dao = this.LRUCachingDAO.create({
+            dao : this.mdao,
```

Comment:
I think we could specify a factory on the 'dao' property of LRUCachingDAO so it would be easier to use. Setting it to an MDAO with 'of' being the same as the 'of' of the delegate is what users will do 99% of the time, so we should make this default.